utility/src/main/java/org/oppia/util/networking/NetworkConnectionUtil.kt

```diff
@@ -0,0 +1,41 @@
+package org.oppia.util.networking
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+import androidx.annotation.VisibleForTesting
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Utility to get the current connection status of the device. */
+@Singleton
+class NetworkConnectionUtil @Inject constructor(private val context: Context) {
+  enum class ConnectionStatus {
+    LOCAL, // Connected to WIFI or Ethernet
```

Comment:
Nit: this should be a KDoc on the constant rather than a single line comment. Ditto for others.

Reply:
Updated.